api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -6142,3 +6143,16 @@ message SessionRecordingConfigUpdate {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// AccessPathChanged is emitted when access graph detects a change in a access path.
+message AccessPathChanged {
+  // Metadata is a common event metadata
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // ChangeID is the id of the change.
+  string ChangeID = 2 [(gogoproto.jsontag) = "change_id"];
```

Comment:
should we include more details in the audit log entry like affected resources,... so users can search them

Reply:
Added